### PR TITLE
feat: IO timers

### PR DESCRIPTION
**Pull request by victorapm**
Add timing statistics for output file writing. Can be turned on with `logLevel >= 1`

Useful for SPE11 tests and benchmarks in general.

### CODE REVIEW

**Comment by victorapm** [comment]
Rebaseline required due to new parameter:

```
  ********************************************************************************
  Error: /Problem/Outputs/siloOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
  ********************************************************************************
  Error: /Problem/Outputs/restartOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
  ********************************************************************************
  Error: /Problem/Outputs/siloOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
  ********************************************************************************
  Error: /Problem/Outputs/restartOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
  ********************************************************************************
  Error: /Problem/Outputs/restartOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
  ********************************************************************************
  Error: /Problem/Outputs/siloOutput
  	Group has a child 'logLevel' in the file to compare but not the baseline file.
  ********************************************************************************
```

**Review by untereiner — APPROVED** [review]

File: src/coreComponents/fileIO/Outputs/OutputBase.hpp [hunk 1]
```diff
@@ -21,11 +21,34 @@
...
+   * @brief Get the minimum log level for output timing information
+   * @return The minimum log level
+   */
+  static constexpr int getMinLogLevel() { return 1; }
+
+  /**
+   * @brief Get the description of this timing category
+   * @return Description string for output timing information
+   */
+  static constexpr std::string_view getDescription() { return "Output timers information"; }
```
Comment (paveltomin) [inline src/coreComponents/fileIO/Outputs/OutputBase.hpp]: since it only works for VTK, it should be placed in VTK output based on my understanding

Comment (victorapm) [reply src/coreComponents/fileIO/Outputs/OutputBase.hpp]: Yes, let me fix this

Comment (victorapm) [reply src/coreComponents/fileIO/Outputs/OutputBase.hpp]: I actually implemented timers for all possible outputs in [7d8677d](https://github.com/GEOS-DEV/GEOS/pull/3480/commits/7d8677d89139f7d7c745c84ab91019eb1cdf8436)

Comment (paveltomin) [reply src/coreComponents/fileIO/Outputs/OutputBase.hpp]: perfect, thanks

**Review by paveltomin — APPROVED** [review]

**Pull request merged**